datarepo-clienttests/src/main/java/utils/BulkLoadUtils.java

```diff
@@ -135,22 +94,33 @@ private BulkLoadArrayRequestModel buildBulkLoadFileRequest() {
     return arrayLoad;
   }
 
-  private void getAndDisplayResults(RepositoryApi repositoryApi, JobModel bulkLoadArrayJobResponse)
+  public BulkLoadArrayRequestModel buildSecondArrayLoadTry() {
+    return new BulkLoadArrayRequestModel().profileId(billingProfileModel.getId()).loadTag(loadTag);
+  }
+
+  public void getAndDisplayResults(RepositoryApi repositoryApi, JobModel bulkLoadArrayJobResponse)
       throws Exception {
     bulkLoadArrayJobResponse =
         DataRepoUtils.waitForJobToFinish(repositoryApi, bulkLoadArrayJobResponse);
+
     BulkLoadArrayResultModel result =
         DataRepoUtils.expectJobSuccess(
             repositoryApi, bulkLoadArrayJobResponse, BulkLoadArrayResultModel.class);
 
     BulkLoadResultModel loadSummary = result.getLoadSummary();
+
+    assertThat(
+        "Number of successful files loaded should equal total files.",
+        loadSummary.getTotalFiles(),
+        equalTo(loadSummary.getSucceededFiles()));
     logger.debug("Total files    : {}", loadSummary.getTotalFiles());
     logger.debug("Succeeded files: {}", loadSummary.getSucceededFiles());
     logger.debug("Failed files   : {}", loadSummary.getFailedFiles());
     logger.debug("Not Tried files: {}", loadSummary.getNotTriedFiles());
   }
 
   public void cleanup(Map<String, ApiClient> apiClients) throws Exception {
+    KubernetesClientUtils.changeReplicaSetSizeAndWait(1);
```

Comment:
Could you move this to the testscript classes instead of in the utils class, since it's not really bulkload related?